src/test/java/core/util/HOCurrencyTests.java

```diff
@@ -0,0 +1,33 @@
+package core.util;
```

Comment:
I am intrigued as to why you didn't write the tests in Kotlin? :-)

Reply:
Step by step - first i want to see, how to use it in java code ;-)

I will change this.